deployer/run.sh

```diff
@@ -83,7 +83,7 @@ metric_resolution=${METRIC_RESOLUTION:-30s}
 project=${PROJECT:-openshift-infra}
 
 # the master certificate and service account tokens
-master_ca=${MASTER_CA:-/var/run/secrets/kubernetes.io/serviceaccount/ca.crt}
+master_ca=${MASTER_CA:-/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt}
```

Comment:
Confirm with @mwringe if this is correct. IIRC, he mentioned that this might not be appropriate.